scripts/uberenv/project.json

```diff
@@ -4,7 +4,7 @@
 "package_final_phase" : "hostconfig",
 "package_source_dir" : "../..",
 "spack_url": "https://github.com/spack/spack",
-"spack_commit": "ccf94ded67e688770dc8d0388d8f75d0d6641e1e",
+"spack_commit": "e4c1cb380beb088393e47ba9caa27a2e5953b4db",
```

Comment:
@white238 -- I'm assuming this is intentional, but why is the `spack_commit` for `devtools.json` different from the one for `project.json` ?

Reply:
I don't want to rebuild all the devtools but I need a high enough SHA that to work with the new `packages.yaml` format.  `devtools.json` was bumped to a known good SHA (I use that one in Serac).  The main one, `project.json`, is being bumped to the newest SHA because I am running into build failures.